### PR TITLE
Use new batch runner config

### PR DESCRIPTION
**Pull request by jcox-dev**
Depends on https://github.com/ONSdigital/census-rm-acceptance-tests/pull/93

Uses release config (in k8s repo) of batch runner for prod-targeting release pipeline.

### CODE REVIEW

**Review by liamtoozer — APPROVED** [review]

File: pipelines/manual-ci-kubernetes-pipeline.yml [hunk 1]
```diff
@@ -1,5 +1,13 @@
 ---
 resources:
+
+- name: acceptance-tests-repo
+  type: git
+  source:
+    uri: git@github.com:ONSdigital/census-rm-acceptance-tests.git
+    private_key: ((github.service_account_private_key))
+    paths: [kubernetes.env, tasks/*]
```
Comment (AdamHawtin) [inline pipelines/manual-ci-kubernetes-pipeline.yml]: This seems to be our releases pipeline now, do we still want the acceptance tests in it?

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
We should probably rename the `manual-ci-kubernetes-pipeline.yml` since it's morphed into a releases pipeline. The README needs updating also.

File: pipelines/manual-ci-kubernetes-pipeline.yml (hunk 1 above)
Comment (jcox-dev) [reply pipelines/manual-ci-kubernetes-pipeline.yml]: @AdamHawtin @davidmort said we're right to keep the acceptance tests in for now.

**Comment by jcox-dev** [comment]
@AdamHawtin renamed and updated readme.

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**